clinica/iotools/bids_utils.py

```diff
@@ -401,9 +401,19 @@ def create_scans_dict(
                     glob.glob(file_to_read_path)[0], sheet_name=sheet
                 )
             elif file_ext == ".csv":
-                file_to_read = pd.read_csv(
-                    glob.glob(file_to_read_path)[0], sep=None, engine="python"
-                )
+                if study_name == "AIBL":
+                    file_to_read = pd.read_csv(
+                        glob.glob(file_to_read_path)[0],
+                        sep=None,
+                        usecols=list(range(0, 36)),
```

Comment:
Usually it's a good thing to avoid magic numbers like this. I'm afraid we will forget why this column filtering was implemented at this specific value for AIBL.
One solution is to read only the columns we need (which seems to be the same for all studies?), rather than reading them all.
If this is not possible for some reason, maybe we can have a small comment explaining why we are filtering and linking the URL of the issue. WDYT?

Reply:
alright. I'll. try to figure out what columns we need though it is unclear to me in the code ! Otherwise i'll leave it as it is and link to the issue or this PR